src/config/SPRACINGH7RF/config.h

```diff
@@ -129,7 +129,7 @@
 #define CURRENT_METER_2_ADC_INSTANCE  ADC3
 #define CURRENT_METER_1_ADC_PIN       PC1
 #define CURRENT_METER_1_ADC_INSTANCE  ADC3
-#define EXTERNAL1_ADC_PIN             PC2
+#define ADC_EXTERNAL1_PIN             PC2
 #define EXTERNAL1_ADC_INSTANCE        ADC3
```

Comment:
I think this should also be ADC_EXTERNAL1_INSTANCE

Reply:
Thanks. Fixed.